src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/classPath_pending.jelly

```diff
@@ -0,0 +1,74 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+    <div class="pane-frame">
+        <p id="pendingClasspathEntries-none">
+            No pending classpath entry approvals.
+        </p>
+        <div id="pendingClasspathEntries">
+            Classpath entries pending approval. (Beware of remote URLs, workspace files, or anything else that might change without your notice.)
```

Comment:
```suggestion
            Classpath entries pending approval. (Beware of remote URLs, workspace files, or anything else that might change without you noticing.)
```

I'm not sure if this is what it means....

`change without notice` is a technical thing, and I'm not sure it calls to mind the right concern.